src/components/hooks/useShareControl.js

```diff
@@ -0,0 +1,29 @@
+import {useContext, useEffect, useState} from 'react';
+
+import {AdapterContext} from '../../components/';
+
+/**
+ * Custom hook that returns the display state of the meeting share control.
+ *
+ * @param {string} meetingID  ID of the meeting for which to get data
+ * @returns {shareControlDisplay}  Data state of the share control
+ */
+export default function useShareControl(meetingID) {
```

Comment:
Not sure I understand the purpose of this hook? `screen-share` should work like any other control and be useable through the [existing `useMeetingControl` hook](https://github.com/webex/components/blob/master/src/components/hooks/useMeetingControl.js)

Reply:
Will reuse the useMeetingControl().